tests/scripts/run-throughput-perf.py

```diff
@@ -200,7 +202,7 @@ def validate_arg(arg, check):
         benchview_path = os.path.normpath(benchview_path)
         validate_arg(benchview_path, lambda item: os.path.isdir(benchview_path))
 
-    args = (arch, operating_system, os_group, build_type, run_type, clr_root, assembly_root, benchview_path, iterations, opt_level, jit_name)
+    args = (arch, operating_system, os_group, build_type, run_type, clr_root, assembly_root, benchview_path, iterations, opt_level, jit_name, no_pgo)
 
     # Log configuration
     log('Configuration:')
```

Comment:
I think we should include whether PGO is on/off in the log here

Reply:
Sure thing. Good catch!